examples/machine-set.yaml

```diff
@@ -6,8 +6,6 @@ metadata:
   namespace: test
   labels:
     machine.openshift.io/cluster-api-cluster: tb-asg-35
-    machine.openshift.io/cluster-api-machine-role: infra
-    machine.openshift.io/cluster-api-machine-type: master
 spec:
```

Comment:
Machine HC controller is still using the labels to filter out master nodes [1]. Do we have any replacement for it? 

[1] https://github.com/openshift/machine-api-operator/blob/c4a348a011c07db7e3f864380b0134b318112d19/pkg/controller/machinehealthcheck/machinehealthcheck_controller.go#L335-L336

Reply:
will update mao as well. Thanks!

Reply:
machine HC already validates from node role https://github.com/openshift/machine-api-operator/blob/c4a348a011c07db7e3f864380b0134b318112d19/pkg/controller/machinehealthcheck/machinehealthcheck_controller.go#L340-L360